client/components/footer/styles.js

```diff
@@ -0,0 +1,21 @@
+import styled from "styled-components";
+
+export const Wrapper = styled.div`
+  width: 100%;
+  background-color: #333;
+  height: 66px;
+  display: flex;
+  align-items: center;
+`;
+
+export const ContentWrapper = styled.div`
+  width: 1440px;
```

Comment:
You can use the pre-defined max-width from the frontend-config for this one :)

Reply:
@Soreg Aye, should be resolved now